modules/run/artifacts/opt/jboss/container/java/run/run-env.sh

```diff
@@ -1,6 +0,0 @@
-#!/bin/sh
-
-# Default the application dir to the S2I deployment dir
-if [ -z "$JAVA_APP_DIR" ]
```

Comment:
note that this script did not assign to JAVA_APP_DIR if it was already set

Reply:
ok